issuetracking.md

```diff
@@ -0,0 +1,57 @@
+# Issue tracking
+The vstest project tracks issues and feature requests using the [issue template](https://github.com/Microsoft/vstest/blob/master/.github/ISSUE_TEMPLATE.md) for the vstest repository.
+
+## Submitting an Issue
+First, please do a search in [issues](https://github.com/Microsoft/vstest/issues) to see if the issue or feature request has already been filed. Use this [query](https://github.com/Microsoft/vstest/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc) to search for the most popular feature requests.
+If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment.
+- 👍 - upvote
+- 👎 - downvote
+
+If you cannot find an existing issue that describes your issue, submit it using the [issue template](https://github.com/Microsoft/vstest/blob/master/.github/ISSUE_TEMPLATE.md). Remember to follow the instruction mentioned therein carefully.
+
+## Issue triage
+Please follow the guidelines in the issue template when filing an issue or a pull request
+New issues or pull requests submitted by the community are triaged by a team member using the inbox query.
+
+## Inbox query
+The [inbox query](https://github.com/Microsoft/vstest/issues?utf8=%E2%9C%93&q=is%3Aopen%20no%3Aassignee%20-label%3Abacklog%20-label%3Aenhancement) will contain all the
+- Open issues or pull requests that 
+- Are not enhancements and
+- Have no owner assigned
+
+## Initial triage - Issue tagging
+Issues will then be tagged as follows:
+- Is the issue ***invalid***? It will be closed, and the reason will be provided.
```

Comment:
For Invalid case should it be more like a note at the end as there is no tag involved for this one?

I am a little skeptical using the word Invalid here, because if I am filing an issue when will it be invalid? we might want to add an example.

Some other cases I could think of 
a. Might not belong to this repo.
b. Might need more info.
c. Has already been fixed.